mjidris/irondb
# Us 203

## Objective
Added the frontend of the login. 

## Detailed Description
This is the most recent version of the login page that is correctly centered. It does not include the backend postgres login. Currently it will flip the isAuthenticated variable but the routes are not re-rendered so the navbar will not change but if you attempt to access the login page again it will show you as already being logged in.

## Screenshot (if applicable)
![image](https://user-images.githubusercontent.com/48575921/73128000-9ecb7680-3f96-11ea-8372-e167a375ac7a.png)


## How to Test (if applicable)

1. Click on "Sign-in" on top right.
2. Login with any fake credentials that comply with minimum password requirements
3. You will be informed via alertbox that you are logged in (Placeholder)
4. Click "Sign-in" again to receive a notice that you are already logged in


## Tests (if added)
NA

## Requirements
[Taiga User Story](https://tree.taiga.io/project/cbrobsto-iron-db/us/174)
[Taiga Task](https://tree.taiga.io/project/cbrobsto-iron-db/task/203)
